supplementary_style_guide/style_guidelines/grammar.adoc

```diff
@@ -103,4 +103,28 @@ The following list includes homographs that might commonly appear in technical d
 * object
 * project
 
+[[minimalism]]
+== Minimalism
+Minimalism is a methodology for creating targeted documentation focused on your readers' needs. If you understand your customers' needs, you can write shorter and simpler documentation specific to what customers want to do. For more information about minimalism, see link:https://source.redhat.com/groups/public/minimalism-quality-initiative[Minimalism in Red Hat documentation].
+
+There are five principles of minimalism:
+
+=== Customer focus and action orientation
+Know what your users do, what their goals are, and why they perform these actions. Minimize how much content customers must wade through to get to something they recognize as real work. Separate conceptual and background information from procedural tasks. 
+
+=== Findability
+Findability covers two areas:
+
+* Make sure your content is findable through Google search and access.redhat.com site searches.
```

Comment:
In the IBM style guide, I think it is recommended that we avoid using phrasal verbs. Does the same guideline apply to this style guide?
```suggestion
* Ensure your content is findable through Google search and access.redhat.com site searches.
```

Reply:
Made suggested change